README.md

```diff
@@ -25,9 +24,11 @@ It's important to know how to compare the tests against each other and what thos
 * Docker for macs is slow on file operations compared to linux
 * Sometimes it's possible that one of the clouds will fail on some operations. Keep in mind that it's not valid to compare a test with failures against a test where all checks are green
 
-**Test setup at ownCloud:**
+**Usage at ownCloud:**
+
+Here at ownCloud we use k6 to compare the performance of our cloud systems, also during development it is helpful to see how the changes made affect performance,
```

Comment:
At ownCloud k6 is used to compare the performance of the products during development. It is very helpful to understand how changes to the codebase affect the performance, between releases, but also between single commits.

Reply:
🥇 thanks, updated